.github/workflows/release-drafter-fiberzerolog.yml

```diff
@@ -0,0 +1,19 @@
+name: Release Drafter Fiberzerolog
```

Comment:
The tests.yml has to be modified to include the tests from this middleware.

Dependabot.yml also has to be updated

Reply:
done! also included in `security.yml` :)